djadmin2/views.py

```diff
@@ -2,9 +2,11 @@
 from __future__ import division, absolute_import, unicode_literals
 
 import operator
+from functools import reduce
```

Comment:
Actually it's used there, see views.py line 163:
https://github.com/galuszkak/django-admin2/blob/tox_tests/djadmin2/views.py#L163